features/org.wso2.carbon.kernel.feature/resources/bin/carbon.sh

```diff
@@ -100,6 +101,10 @@ if $mingw ; then
   # TODO classpath?
 fi
 
+#Set the runtimehome based on carbon.sh location
+TEMPCURDIR=`dirname "$PRG"`
+RUNTIME_HOME=`cd "$TEMPCURDIR/.." ; pwd`
```

Comment:
@thusithathilina / @daneshk : here we are setting the runtime as a system property from the carbon.sh in the runtime. Did we test this if we run only the runtime carbon.sh from different folder location?